### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by Lisand7ru**
Sorry for new pull request

### CODE REVIEW

File: lib/telegraf.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ class Telegraf extends Composer {
     }
     this.telegram.getUpdates(timeout, limit, offset, allowedUpdates)
       .catch((err) => {
-        const wait = err.retryAfter || this.options.retryAfter
+        const wait = err.parameters.retry_after || this.options.retryAfter
```
Comment (dotcypress) [inline lib/telegraf.js]: I think we need to add an extra check:
`(err.parameters && err.parameters.retry_after) || this.options.retryAfter`

Comment (Lisand7ru) [inline lib/telegraf.js]: TelegramError.parameters always contains an object
`this.parameters = payload.parameters || {}`

Comment (dotcypress) [inline lib/telegraf.js]: Yes, but we can't claim that all Errors will be instance of `TelegramError`

**Comment by dotcypress** [comment]
Thank you 👍

**Pull request merged**